dist/systemd/system/installer.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+################################################################################
+# SPDX-License-Identifier: Apache-2.0
+################################################################################
+
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root" 1>&2
+   exit 1
+fi
+
+BASEDIR=$(dirname "$0")
+
+# check keylime agent location
+KEYLIMEDIR=$(dirname $(which keylime_agent | cut -d " " -f 2))
+if [[ $KEYLIMEDIR == "." ]]; then
+    echo "Unable to find keylime agent" 1>&2
+    exit 1
+fi
+
+echo "Using keylime directory: ${KEYLIMEDIR}"
+
+# prepare keylime service files and store them in systemd path
+sed "s|KEYLIMEDIR|$KEYLIMEDIR|g" $BASEDIR/keylime_agent.service.template > /etc/systemd/system/keylime_agent.service
+
+# Copy secure mount
```

Comment:
Why not extend the existing [GNUmakefile](https://github.com/keylime/rust-keylime/blob/master/GNUmakefile) and use `install` rather than manual calls to `chmod` and `chown`?

Reply:
Because I did it without thinking much :)
I will make the changes in the Makefile.

Reply:
Removed the installer script completely and added minimal changes to the `GNUmakefile`.